stubs/Markdown/markdown/util.pyi

```diff
@@ -16,7 +16,7 @@ AMP_SUBSTITUTE: str
 HTML_PLACEHOLDER: str
 HTML_PLACEHOLDER_RE: Pattern[str]
 TAG_PLACEHOLDER: str
-RTL_BIDI_RANGES: Any
+RTL_BIDI_RANGES: tuple[tuple[str, str], tuple[str, str]]
```

Comment:
Optional: This, and the other constants before it should probably be marked `Final`.